packages/core/src/pages/[...slug].tsx

```diff
@@ -100,19 +106,35 @@ export const getStaticProps: GetStaticProps<
     }
   }
 
-  const [{ data, errors = [] }, page] = await Promise.all([
+  const [{ data, errors = [] }] = await Promise.all([
     execute<
       ServerCollectionPageQueryQueryVariables,
       ServerCollectionPageQueryQuery
     >({
       variables: { slug },
       operation: query,
     }),
-    getPage<PLPContentType>({
+  ])
+
+  let pageData
+
+  if (storeConfig.cms.data) {
+    const cmsData = JSON.parse(storeConfig.cms.data)
+    const page = cmsData['plp'][0]
```

Comment:
What I mentioned about the template in the platform's PR is directly related to this line. Since we started to implement the Multiple page template approach. So if we have the template here, we can also match the right page, instead of getting the first.